server/server-kv.c

```diff
@@ -46,21 +92,26 @@ handle_request(void *ptr)
 		pthread_mutex_unlock(&mutex);
                 clientfd = node->fd;
 
-                //for test
-                bytes_read = read(clientfd, message, 256);
-		if(bytes_read < 0) {
-			perror("ERROR reading socket");
-		}
-		else if (bytes_read == 0){
-			printf("Client disconnected");
-		}
-		else {
-                        strcpy(message, gwkv_handle_operation(ht, message));
-                        bytes_write = strlen(message)+1;
-			write(clientfd, message, bytes_write);
-                        printf("thread: %d, received:%s\n", pthread_self(), message);
+		while(1){
```

Comment:
What's the value of adding an infinite loop? Shouldn't this be handled per thread?